src/client/pages/StaffViewAuthPage/index.js

```diff
@@ -0,0 +1,299 @@
+// This page is identical to RetroCertsAuthPage, minus the captcha, Spanish
+// translation button, and helper text below the header. It also has a
+// different error message for userNotFound.
+import { useTranslation } from "react-i18next";
+import { useHistory } from "react-router-dom";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Alert from "react-bootstrap/Alert";
+import React, { useState, useEffect } from "react";
+import AUTH_STRINGS from "../../../data/authStrings";
+import routes from "../../../data/routes";
+import { userDataPropType, setUserDataPropType } from "../../commonPropTypes";
+import Footer from "../../components/Footer";
+import Header from "../../components/Header";
+import SessionTimer from "../../components/SessionTimer";
+import weeksCompleted from "../../../utils/checkFormData";
+import { fromIndexToPathString } from "../../../utils/retroCertsWeeks";
+import Inputmask from "inputmask";
+import { autoScroll, TOP, BEHAVIOR } from "../../../utils/autoScroll";
+
+function StaffViewAuthPage(props) {
+  const { t } = useTranslation();
+  document.title = t("retrocert-login.title");
+  const history = useHistory();
+
+  const userData = props.userData;
+  const setUserData = props.setUserData;
+
+  const [lastName, setLastName] = useState("");
+  const [dobMonth, setDobMonth] = useState("");
+  const [dobDay, setDobDay] = useState("");
+  const [dobYear, setDobYear] = useState("");
+  const [ssn, setSsn] = useState("");
+  const [validated, setValidated] = useState(false);
+  const [showSsn, setShowSsn] = useState(false);
+  const [showGenericValidationError, setShowGenericValidationError] = useState(
+    false
+  );
+
+  const status = userData && userData.status;
+  const errorTransKey = new Map([
+    [
+      AUTH_STRINGS.statusCode.userNotFound,
+      "retrocert-login.invalid-user-error-staff-view",
+    ],
+    [
+      AUTH_STRINGS.statusCode.sessionTimedOut,
```

Comment:
This won't be used on this page, but maybe you're keeping it to reduce the differences with RetroCertsAuthPage?

Reply:
nope not intentional, thanks